### ISSUE TITLE
Add new topic: wifi_ota_message to support pcap formatted message

### ISSUE THREAD

**Issue by glencornell** [christianrowlands/network-survey-messaging#6]
In some cases, it would be useful to have an accommodation in the protocol which provides a generalized method of capturing a single packet for later analysis.  This is not intended to replace more efficient methods of packet capture (such as kismet databases, pcap files, etc.).  Rather, it is intended to capture a specific type of packet in a generalized way while preventing unbounded changes to this protocol.  The proposed change would follow the other OTA messages that encapsulate the packet in pcap format.

Should this proposal be accepted, I am willing to submit the pull request,

**Comment by christianrowlands** [comment]
Yeah, I think this would work. The latest version of Kismet already has a nice API defined for streaming pcap binary records over a socket. I wrote a Java client a while back that parsed those records and displayed them on a map, and the Kismet API was really easy to use. However, if you need MQTT transfered packets, and you only need a few as opposed to an entire pcap file then I think adding it to this protocol makes sense.

**Comment by glencornell** [comment]
I fixed the tests.  However, I had to uncomment the line in the dependencies for javax.annotation-api.  I did not include this diff in my pull request because I think it might break your build environment.  Different versions of gradle, perhaps?

`diff --git a/build.gradle b/build.gradle
index 868dbb9..b4f2fb2 100644
--- a/build.gradle
+++ b/build.gradle
@@ -46,7 +46,7 @@ dependencies {
 
     implementation "com.google.protobuf:protobuf-java:$protobufVersion"
 
-    //implementation "javax.annotation:javax.annotation-api:1.3.2"
+    implementation "javax.annotation:javax.annotation-api:1.3.2"
 
     testImplementation "com.google.protobuf:protobuf-java-util:3.11.4"
     testImplementation 'org.junit.jupiter:junit-jupiter-api:5.3.1'
`

Here's my build environment:

`$ ./gradlew --version

------------------------------------------------------------
Gradle 6.3
------------------------------------------------------------

Build time:   2020-03-24 19:52:07 UTC
Revision:     bacd40b727b0130eeac8855ae3f9fd9a0b207c60

Kotlin:       1.3.70
Groovy:       2.5.10
Ant:          Apache Ant(TM) version 1.10.7 compiled on September 1 2019
JVM:          11.0.15 (Private Build 11.0.15+10-Ubuntu-0ubuntu0.20.04.1)
OS:           Linux 5.14.0-1036-oem amd64

`
